### ISSUE TITLE
Can starganv2 achieve comparable results with faceapp?

### ISSUE THREAD

**Issue by youngstu** [clovaai/stargan-v2#57]
Faceapp is very popular and it's effect is very shocking. Starganv2 shows us the potential to achieve comparable effect of faceapp. I wonder if author has studied faceapp?

**Comment by youngstu** [comment]
Have you studied the design architecture behind faceapp？

**Comment by doantientai** [comment]
I don't think it is fair to compare a paper implementation with completed product.
